src/tracer/src/extracts/ebpf_watcher/watcher.rs

```diff
@@ -295,3 +295,23 @@ impl EbpfWatcher {
             .await
     }
 }
+
+fn get_process_argv(pid: i32) -> Vec<String> {
```

Comment:
maybe this goes to some  process utility class?

Reply:
Do you have a suggestion? I don't see any existing struct/mod where I could move it. Don't really want to create a new one just for a single function, especially since it's only used in this file.